x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_define_rule/translations.tsx

```diff
@@ -62,3 +62,11 @@ export const ML_JOB_SELECT_PLACEHOLDER_TEXT = i18n.translate(
     defaultMessage: 'Select a job',
   }
 );
+
+export const ENABLE_ML_JOB_WARNING = i18n.translate(
+  'xpack.siem.detectionEngine.createRule.stepDefineRule.mlEnableJobWarningTitle',
+  {
+    defaultMessage:
+      'This job is not currently running. Please enable job via "ML job settings" interface before running rule.',
```

Comment:
'This ML job is not currently running. Please set this job to run via "ML job settings" before activating this rule.',